templates/Makefile

```diff
@@ -0,0 +1,22 @@
+# ================================================================================
+# Copyright (c) 2014-2023 VMware, Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the “License”); you may not
+# use this file except in compliance with the License.  You may obtain a copy of
+# the License at:
+#
+#              http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an “AS IS” BASIS, without warranties or
+# conditions of any kind, EITHER EXPRESS OR IMPLIED.  See the License for the
+# specific language governing permissions and limitations under the License.
+# ================================================================================
+
+PREFIX ?= /usr
+DATADIR ?= $(PREFIX)/share/open-vmdk
+
+all:
+
+install:
+	mkdir -p $(DESTDIR)/$(DATADIR)/ && cp *.ovf $(DESTDIR)/$(DATADIR)
```

Comment:
If all OVF templates are installed under share folder, does that mean mkova.sh can find it under ../share/open-vmdk dir? In such case, the mkova.sh command could be more simpler with option like `--hw 15` and then mkova.sh is going to find the real path to template-hw15.ovf. What's your opinion?

Reply:
I think that's a good idea.

Reply:
Done.